pyrs/transpiler.py

```diff
@@ -256,6 +261,11 @@ def visit_Attribute(self, node):
 
         value_id = self.visit(node.value)
 
+        if value_id == "sys":
+            if attr == "argv":
+                self._usings.add("std::env")
+                return "env::args().map(|s| &*Box::leak(s.into_boxed_str())).collect()"
```

Comment:
What's the Box::leak doing? Do you have a reference for this method?

Reply:
https://doc.rust-lang.org/std/boxed/struct.Box.html#method.leak

This is needed to force env::args into a Vec of `str`, which is forced on us at the moment.

Reply:
```
    let __temp1: Vec<String> = env::args().collect();
    let a: Vec<&str> = __temp1.iter().map(|s| s as &str).collect();
```

reads easier. This is fine for now, since generating the above requires additional code in the transpiler.